modoboa/bower_components/swagger-ui/.bower.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "swagger-ui",
```

Comment:
Why this new dependency?

Reply:
It is needed and used to render the API documentation in `modoboa/templates/swagger-ui.html`. It comes with e0c7305149097cb40ef4b32641f93881e12071f3 and is explained in the [Django REST Framework doc](https://www.django-rest-framework.org/topics/documenting-your-api/). The other choice would be [ReDoc](https://github.com/Rebilly/ReDoc), but as Swagger UI is more generally used... Also, this is needed because of the drop of `CoreAPI` - which is needed to use [`include_docs_urls`](https://www.django-rest-framework.org/coreapi/from-documenting-your-api/#include_docs_urls).

Reply:
Ok, makes sense.